testsuite/test_statistics.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env python
+
+"""
+Tests for statistics (`_statistics` module).
+"""
+
+from __future__ import absolute_import, print_function
+
+import time
+import unittest
+
+from pywbem import Statistics
+
+
+def time_abs_delta(t1, t2):
+    """
+    Return the positive difference between two float values.
+    """
+    return abs(t1 - t2)
+
+
+class StatisticsTests(unittest.TestCase):
+    """All tests for Statistics and TimeCapture."""
+
+    def test_enabling(self):
+        """Test enabling and disabling."""
+
+        statistics = Statistics()
+
+        self.assertFalse(statistics.enabled,
+                         "Error: initial state is disabled")
```

Comment:
The message is printed if the assertion fails, so it should say what is not correct. In this case, what is not correct is that the initial state is *not* disabled.